### PR TITLE
Report minor tweeks

### PR DESCRIPTION
**Pull request by RolRodr**
PR addresses 4 tickets:

1. [Min/Max column width](https://unclibrary.atlassian.net/browse/BXC-4282):  Changed image column to 25% of table width since there is no convenient way to set min/max within datatables. If requested, I can modify the css to set min/max, but this might do.

2. [Create CSV link](https://unclibrary.atlassian.net/browse/BXC-4283): Created CSV link button next to table toggle button. Each button has a different background for identification. Happy to adjust if requested.

3. [Keybinding to navigate pages](https://unclibrary.atlassian.net/browse/BXC-4284): Added key bindings for left and right arrow keys to move to previous and next page respectively. Can add additional keybindings if requested.

4. [Loading icon](https://unclibrary.atlassian.net/browse/BXC-4285): Added loading icon for initial load and between toggling tables. The loading icon itself still lags a bit after pressing the toggle button, unsure if this can be addressed.

### CODE REVIEW

**Review by bbpennel — CHANGES_REQUESTED** [review]
The key bindings work great!

I'm getting the loading icon on initial page load, but not for other operations like applying filters or sorting by columns (which tend to take my computer about 2 seconds each). Maybe there's a tiny flicker of the icon when toggling modes right as the new content renders. I'm guessing the browser is lagging and not able to render the icon. Did you try enabling the datatables processing indicator? that might cover the sorting/filtering cases
https://datatables.net/reference/option/processing
https://datatables.net/forums/discussion/63919/loading-text-or-icon-when-data-loads

File: create_report.py [hunk 1]
```diff
@@ -26,26 +26,39 @@
...
+# parser creates item-level data and aggregate data if indicated
 data = parser.get_data()
 stats = False
 if args.agg_data:
     stats = parser.get_stats()
-generator.create_html_page(data, stats)
+
+# create url for csv download
+if args.normalized_url:
+    csv_url = args.normalized_url+str(args.file_path)
```
[the remark this answers is not in the document]
Comment (RolRodr) [reply create_report.py:54]: @bbpennel will the reports always come from the reports directory? I'll just go ahead an make '/shared/reports/' the substring used to normalize the path.

[the remark this answers is not in the document]
Comment (bbpennel) [reply create_report.py:54]: I think the report gets generated as a sibling of the input, so it should be at the same path. But its probably fine to look for /shared/reports/, its not going to moved around much.

I guess if the latency ends up being too major of an issue, we could add a server side component.

[the remark this answers is not in the document]
Comment (RolRodr) [reply create_report.py]: I believe the latest push should fix the csv url, as well as the spinner issue by giving a slight delay when toggling off while being asynchronous with the table toggles. It won't address the latency issue, however. I'm happy to start working on the server side component.

**Review by bbpennel — APPROVED** [review]

**Pull request merged**